data/pages.yaml

```diff
@@ -48,4 +48,43 @@ index:
   maintainer_resources_action: Find maintainer resources
 ecosystem:
   title: Ansible Ecosystem
-  intro: Projects in the Ansible ecosystem let you expand automation to a virtually unlimited set of use cases.
\ No newline at end of file
+  intro: Projects in the Ansible ecosystem let you expand automation to a virtually unlimited set of use cases.
+# Developer page
+developer:
+  title: Developer resources
+  intro: Extend automation with custom Ansible modules, add functionality to existing modules, or fix bugs to improve existing code.
+  milestone:
+    start: Start writing code
+    contribute: Contribute code to an Ansible project
+    test: Test plugins and modules
+    create_collection: Create new collections
```

Comment:
Not sure where all this is controlled in the code but can you make the milestones font bigger so it stands out a bit more?

Reply:
That's in the stylesheet and should be a different PR imo. It would be nice to get a review of all the scss content.

Reply:
I was wrong. it was the heading level that needed changing. thanks